frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -47,6 +47,17 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.DataResiliencyWithResources),
     },
   },
+  {
+    type: 'Dashboards/Card',
+    properties: {
+      tab: 'persistent-storage',
+      position: GridPosition.MAIN,
+      loader: () =>
+        import(
+          './components/dashboard-page/storage-dashboard/capacity-card/capacity-card' /* webpackChunkName: "ceph-storage-capacity-card" */
+        ).then((m) => m.CapacityCardWithResources),
```

Comment:
This needs to be done once above comment is sorted
```suggestion
        ).then((m) => m.default),
```

Reply:
Thanks for the sample code, was initially confused about how to do it... Changes made...